src/main/msp/msp.c

```diff
@@ -1493,7 +1493,11 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
         sbufWriteU8(dst, 0);
 #endif
-
+#if defined(USE_ITERM_RELAX)
```

Comment:
Add a comment indicating this is for 1.42. In older versions there are no comments but we are trying to add this info for future versions.